src/utils/illustrations/checkIllustration.js

```diff
@@ -0,0 +1,67 @@
+import { exists, removeDir } from '@tauri-apps/api/fs';
```

Comment:
Add todo flag on here

```suggestion
/* TODO: This is part of the cloud syncing feature */
import { exists, removeDir } from '@tauri-apps/api/fs';
```